### PR TITLE
Add section about DNF to consumer docs

### PR DESCRIPTION
**Pull request by daviddavis**
refs #1875
https://pulp.plan.io/issues/1875

### CODE REVIEW

**Comment by bmbouter** [comment]
Does it make any sense to make this a 2.16 release note?

File: docs/user-guide/consumer-client/introduction.rst [hunk 1]
```diff
@@ -44,3 +44,12 @@ Troubleshooting with verbose flag
 You can run pulp-consumer commands in verbose mode to get additional information
 in case of a failure. Look at the example usage of verbose flag in the
 :ref:`admin client troubleshooting section <client-verbose-flag>`.
+
+
+DNF Support
+-----------
+
+It should be noted that as of Pulp 3, consumers will no longer be supported in
+Pulp. As such, we have no plans to add support for DNF. For more information,
```
Comment (goosemania) [inline docs/user-guide/consumer-client/introduction.rst]: Does it make sense, just for clarity, to add `in Pulp 2.`?

`As such, we have no plans to add support for DNF in Pulp 2.`

When I quickly read this note, I thought it's only about Pulp 3, but maybe it's just me :)

**Comment by goosemania** [comment]
@bmbouter , I'm not sure what exactly to add to the release notes. We've never supported DNF and now we just decided that we won't add it. This decision is also not strictly attached to the 2.16 release, imo.

**Comment by daviddavis** [comment]
I updated the docs. I neglected to add a release note because I agree with @goosemania but it's not a strong opinion I have.

**Comment by bmbouter** [comment]
+1 to not add a release note. It's not specific to this release. Thank you for pointing that out. Sorry for the delay in my comment.

**Comment by goosemania** [comment]
@daviddavis , I'm not sure `refs` in the commit message is handled correctly by redmine. Maybe `closes` is better?

**Comment by daviddavis** [comment]
I've updated the PR but I am not sure why this is failing.

**Comment by bmbouter** [comment]
ok test

**Comment by bmbouter** [comment]
I'm also not sure why the one test is failing, but it could be a timing problem with the test. I pasted the failure below. I'm rerunning to see if another run makes it green.

```
======================================================================
FAIL: test_increments_last_run (unit.server.db.model.test_dispatch.TestScheduleEntryNextInstance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/.virtualenvs/test/lib/python2.7/site-packages/mock.py", line 1201, in patched
    return func(*args, **keywargs)
  File "/home/jenkins/workspace/unittest-pulp-pr/node-type/f26-np/pulp/server/test/unit/server/db/model/test_dispatch.py", line 906, in test_increments_last_run
    self.assertTrue(now - next_entry.last_run_at < timedelta(seconds=1))
AssertionError: False is not true
```

**Comment by daviddavis** [comment]
Yea, I saw the timing error. I was thinking that maybe the test ran slowly for whatever reason and maybe we should bump up the duration maybe?

**Comment by bmbouter** [comment]
Yeah bumping up the timing to reduce the false positive failure rate would be good. I'm not sure what @Ichimonji10 would suggest.

**Comment by daviddavis** [comment]
I'm tempted to see if it happens again before trying to remedy it.

**Comment by bmbouter** [comment]
That sounds good.

**Comment by daviddavis** [comment]
@bmbouter @goosemania thanks for the reviews

**Pull request merged**